build-logic/ktlint-custom-rules/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+plugins {
+    `java-library`
```

Comment:
Do we need java in KMP?

Reply:
yes, this is needed - we don't distribute this java library, but it's used internally in the build process